tests/integration/tests/conftest.py

```diff
@@ -40,6 +41,7 @@ def fixture_start_stop_cloudbeat(k8s, api_client, cloudbeat_agent):
                           name=cloudbeat_agent.name,
                           status_list=['ADDED', 'MODIFIED'],
                           namespace=cloudbeat_agent.namespace)
+    time.sleep(5)
```

Comment:
Added to resolve flakiness of `k8s.wait_for_resource`
@gurevichdmitry thanks

Reply:
The flakiness might be related to the fact that now the cluster has 2 pods and not just one.
The `wait_for_resource` function returns after the first find.
I think we need something similar to `kubectl rollout status ds NAME` (https://github.com/kubernetes/kubectl/blob/master/pkg/polymorphichelpers/rollout_status.go#L95)

Reply:
please add an explanation in the code for why we need this wait, so we'll know how to handle it in the future if we want to remove it. Alternatively, we can use the Kubernetes SDK to list all the pods with elastic-agent in their names and wait for the number to be equal to the number of nodes we have, but this is a bit more complicated than this.